src/mpyl/utilities/docker/__init__.py

```diff
@@ -74,7 +92,7 @@ def execute_with_stream(
 ):
     if multiprocess:  # Logger settings need to be re-applied in each process
         logger.setLevel(logging.INFO)
-        logger.addHandler(RichHandler())
+        logger.handlers.clear()
```

Comment:
Why this change? Did you check the difference of the logging output of the cypress step (that's the one that uses multiprocessing)?

Reply:
Yeah, I checked the output and the problem was that there was a nested logger, which messed up the output. Did you see this too?
Should have taken a screenshot...
I guess you can also just kick off a build and check the logs.

Reply:
Ah that would explain a lot! Where did you fix the nested logger in this pr? I didn't notice it during my review

Reply:
This _is_ the fix for the nesting problem. I think `addHandler` was the culprit.

Reply:
Oh... it's been a while, but I swear this was needed to get any sort of formatted logging from the cypress step. Anyway, I'm happy if it works now